include/faabric/mpi/mpi.h

```diff
@@ -86,7 +90,7 @@ extern "C"
 
     struct faabric_request_t
     {
-        int _unused;
+        int id;
```

Comment:
Refactor for better readability as in `faasm` we make no difference between a pointer to an object, and a pointer to an object's first attribute (same address, but in faasm it's hidden under a layer of indirection, hence the first propertie's name does not really matter).

Reply:
Hmm I'm not sure this is any clearer, is this really an `id`? `id` suggests to me that it's an identifier, and somehow assigned by the application to have some meaning. It seems to me more like a `self` pointer, so maybe `_self` is an appropriate name?